R/pkg/R/generics.R

```diff
@@ -942,6 +942,14 @@ setGeneric("countDistinct", function(x, ...) { standardGeneric("countDistinct")
 #' @export
 setGeneric("crc32", function(x) { standardGeneric("crc32") })
 
+#' @rdname create_array
```

Comment:
actually you are right - I saw `###################### Column Methods ##########################` and thought that's the place but you are right, we already have them in both places.

I'm fine with what you have